scripts/generate_airport_codes.js

```diff
@@ -0,0 +1,262 @@
+/*
+ * Script: generate_airport_codes.js
+ * Description: Reads CSV at src/common/app_data/airport_codes.csv and outputs
+ * JSON to common/app_data/airport_codes.json in the required structure.
+ */
+
+const fs = require('node:fs');
+const path = require('node:path');
+const crypto = require('node:crypto');
+
+const CSV_PATH = path.join(__dirname, '..', 'src', 'common', 'app_data', 'airport_codes.csv');
+const OUT_PATH = path.join(__dirname, '..', 'src', 'common', 'app_data', 'airport_codes.json');
+
+function normStr(v) {
+  if (v === null || v === undefined) return '';
+  return String(v).trim().toUpperCase();
+}
+
+function buildId(iata, icao, otherCode) {
+  const payload = `${normStr(iata)}${normStr(icao)}${normStr(otherCode)}`;
+  return crypto.createHash('sha256').update(Buffer.from(payload, 'utf8')).digest('hex');
+}
+
+function toBool(v) {
+  const s = String(v ?? '').trim().toLowerCase();
+  if (s === 'true') {
+    return true;
+  }
+  return false;
```

Comment:
```suggestion
return String(v).trim().toLowerCase() === 'true';
```

Stringified nullish values won't pass the return condition so no need to make them empty strings first